hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -664,11 +664,9 @@
       name: chip_sw_clk_idle_trans
       desc: '''Verify the ability to turn off the transactional clock via SW.
 
-            Ensure that activity in any of the IPs running on this clock prevents the clock from
-            actually being turned off until the activity is completed, at which point the clock is
-            turned off.  Verify it is off via spinwait in hints_status CSR, and then check the state
-            of the transactional unit to correspond to a completed activity.  Verify that turning
-            off this clock does not affect the other derived clocks.
+            Ensure that the clock to transactional units can be turned off regardless of any
```

Comment:
Note that these are already X-ref'ed with other IP tests that generated the idle. So the cost of accomplishing this testpoint is zero.